src/top-bar/context-select/context-select.module.css

```diff
@@ -0,0 +1,27 @@
+.button {
+    all: unset;
+    display: inline-flex;
```

Comment:
Opinion: I think that siblings should not arrange their layout, imo that's the responsibility of the parent element. In this case that'd mean that the `TopBar` would have to wrap its children with an element and use sth like `display: flex`.
It's not an issue in this case because you've set the elements to be inline.